src/components/CodeBlock/CodeBlock.stories.ts

```diff
@@ -0,0 +1,38 @@
+import { CodeBlock } from "./CodeBlock";
+
+export default {
+  title: "CodeBlocks/CodeBlock",
+  component: CodeBlock,
+  tags: ["code-blocks", "code-block", "autodocs"],
+};
+
+export const Playground = {
+  args: {
+    children: `
+    SELECT
```

Comment:
Probably want to remove the space from before SELECT so it's inline with FROM, WHERE etc